accumulo-handler/src/java/org/apache/hadoop/hive/accumulo/mr/HiveAccumuloTableOutputFormat.java

```diff
@@ -60,6 +62,38 @@ public void checkOutputSpecs(FileSystem ignored, JobConf job) throws IOException
     return super.getRecordWriter(ignored, job, name, progress);
   }
 
+  @Override
+  public FileSinkOperator.RecordWriter getHiveRecordWriter(
+      JobConf job, Path finalOutPath, Class<? extends Writable> valueClass, boolean isCompressed,
+      Properties tableProperties, Progressable progress) throws IOException {
+    configureAccumuloOutputFormat(job);
+
+    try {
+      return new MyRecordWriter(job);
+    } catch (AccumuloException | AccumuloSecurityException e) {
+      throw new IOException("Failed to acquire Accumulo DelegationToken", e);
+    }
+  }
+
+  static private class MyRecordWriter extends AccumuloRecordWriter implements
+      RecordWriter<Text, Mutation>, FileSinkOperator.RecordWriter {
```

Comment:
Isn't the `RecordWriter<Text, Mutation>` duplicative since `AccumuloRecordWriter` already implements that?